content/en/storage-providers/advanced-configurations/sealing.md

```diff
@@ -0,0 +1,244 @@
+  DisableCollateralFallback = false
+  # enable / disable precommit batching (takes effect after nv13)
+  BatchPreCommits = true
+  # maximum precommit batch size up to 256 sectors - batches will be sent immediately above this size
+  MaxPreCommitBatch = 256
+  # how long to wait before submitting a batch after crossing the minimum batch size
+  PreCommitBatchWait = "24h0m0s"
+  # time buffer for forceful batch submission before sectors/deal in batch would start expiring
+  PreCommitBatchSlack = "3h0m0s"
+
+  # enable / disable commit aggregation (takes effect after nv13)
+  AggregateCommits = true
+  # minimum batched commit size, no less than 4
+  MinCommitBatch = 4
+  # maximum batched commit size up to 819 sectors - batches will be sent immediately above this size
+  MaxCommitBatch = 819
+  # how long to wait before submitting a batch after crossing the minimum batch size
+  CommitBatchWait = "24h0m0s"
+  # time buffer for forceful batch submission before sectors/deals in batch would start expiring
+  CommitBatchSlack = "1h0m0s"
+
+  # network BaseFee below which to stop doing commit aggregation, instead submitting proofs to the chain individually
+  AggregateAboveBaseFee = 0.00000000015 #0.15nano
+
+
+  TerminateBatchMax = 100
+  TerminateBatchMin = 1
+  TerminateBatchWait = "5m0s"
+```
+
+### PreCommitSectorsBatch
+
+`PreCommitSectorsBatch` introduced by [FIP-0008 ](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0008.md) supports a miner to pre-commit a number of sectors at once.
+
+In lotus v1.10.0 and up, if `BatchPreCommit` is set to false, pre-commitments will be sent to the chain via `PreCommitSector` messages once they are ready. If `BatchPreCommit` is set to true, lotus will batch pre-commitments until any of `MaxPreCommitBatch`, `PreCommitBatchWait` or `PreCommitBatchSlack` is hit:
+- `MaxPreCommitBatch` is the maximum amount of sectors' pre-commitments to batch in one `PreCommitSectorsBatch` message. According to FIP-0008, this values is up to 256.
+- `PreCommitBatchWait` is how long to wait before submitting the current batch. Note: the ticket of pre-commitment has an expiration of approximately **31.5 hours**, one sector's pre-commit ticket expires **WILL** cause the whole message to fail. Therefore, **we recommend miners to set this value lower than 30 hours.**
+- `PreCommitBatchSlack` is the time buffer to forcefully submit the current batch before any of the sector's pre-commit ticket or a deal will expire. For example, if this value is set to 1 hour, which is 120 epochs, then a `PreCommitSectorsBatch` message will be submitted for the existing batch 120 epochs before the earliest epoch among precommits' tickets and deal's start epochs in this batch. **We recommend you to set a longer slack to prevent message failures due to expirations.**
+
+> Note, the current batch will be sent if any of `MaxPreCommitBatch`, `PreCommitBatchWait` or `PreCommitBatchSlack` is hit.
```

Comment:
If you need to add a call out use theme alerts:

```plaintext
{{< alert icon="👉" >}}
_Markdown_ and <em>HTML</em> will be rendered.
{{< /alert >}}
```

Do not use quote blocks for call outs. Quote blocks are for quotes.